indico/modules/events/fields.py

```diff
@@ -62,3 +65,32 @@ def _value(self):
             return []
         else:
             return [{'id': r.id, 'type': unicode(r.reference_type_id), 'value': r.value} for r in self.data]
+
+
+class EventPersonLinkListField(PersonLinkListFieldBase):
+    """A field to manage event's chairpersons"""
+
+    person_link_cls = EventPersonLink
+    widget = JinjaWidget('events/forms/event_person_link_widget.html')
+
+    def __init__(self, *args, **kwargs):
+        self.allow_submitters = kwargs.pop('allow_submitters', True)
```

Comment:
If we never override the defaults why not just pass hardcoded values for those kwargs?